ts/boot/__tests__/configureStoreAndPersistor.test.ts

```diff
@@ -449,5 +449,70 @@ describe("configureStoreAndPersistor", () => {
         expect(globalStateAt46).toEqual(basePersistedGlobalStateAt45);
       })
     );
+    // Test for 46 to 47
+    it("should migrate from 46 to 47", () => {
+      const persistedStateAt46 = {
+        content: {
+          idps: remoteUndefined,
+          municipality: {
+            codiceCatastale: pot.none,
+            data: pot.none
+          },
+          contextualHelp: pot.none
+        },
+        crossSessions: {
+          hashedFiscalCode:
+            "6494e783ad296f016b2105f8fe7dc2979551a37d3a5c40624e2ee8eee64e8017"
+        },
+        installation: {
+          isFirstRunAfterInstall: false,
+          appVersionHistory: [
+            "2.82.0.7",
+            "2.83.0.7",
+            "3.0.0.8",
+            "3.0.1.0",
+            "3.1.0.2",
+            "3.2.0.8",
+            "3.3.0.8",
+            "3.4.0.5",
+            "3.5.0.8",
+            "3.6.0.9"
+          ]
+        },
+        onboarding: {
+          isFingerprintAcknowledged: false
+        },
+        persistedPreferences: {
+          isFingerprintEnabled: undefined,
+          preferredCalendar: undefined,
+          preferredLanguage: undefined,
+          wasServiceAlertDisplayedOnce: false,
+          isPagoPATestEnabled: false,
+          isCustomEmailChannelEnabled: pot.none,
+          continueWithRootOrJailbreak: false,
+          isMixpanelEnabled: null,
+          isPnTestEnabled: false,
+          isIdPayTestEnabled: false,
+          fontPreference: "comfortable",
+          isExperimentalDesignEnabled: false,
+          useMessagePaymentInfoV2: false
+        },
+        profile: pot.none,
+        _persist: {
+          version: 46,
```

Comment:
This `version` is correct. Can you also fix those at:
- line 370 with `44`?
- line 434 with `45`?